src/pages/index.js

```diff
@@ -90,18 +92,71 @@ const DescriptiveText = styled(P)`
     color: ${colorMap.greyLightTwo};
     margin-bottom: ${size.medium};
 `;
+
+const ThumbnailCard = styled(Card)`
+    position: relative;
+`;
+
+const ThumbnailButton = styled(Button)`
+    left: 40%;
+    position: absolute;
+    top: 35%;
+`;
+
+const Thumbnail = ({ video }) => {
+    let thumbnailUrl = greenPatternImage; //fallback image if there is none
+    let dimensionsMatcher = '{width}x{height}';
+    if (video.thumbnail_url) {
+        const containsSpace = video.thumbnail_url.match('%{width}x%{height}');
+        dimensionsMatcher = containsSpace
+            ? '%{width}x%{height}'
+            : dimensionsMatcher;
+        thumbnailUrl = video.thumbnail_url.replace(
+            dimensionsMatcher,
+            `${MEDIA_WIDTH}x${MEDIA_WIDTH}`
+        );
+    }
+
+    return (
+        <ThumbnailCard maxWidth={MEDIA_WIDTH} image={thumbnailUrl}>
+            <Modal
+                dialogContainerStyle={{
+                    height: '90%',
+                    padding: `0 ${size.large}`,
+                    width: '90%',
+                }}
+                transparent
+                triggerComponent={<ThumbnailButton play />}
+            >
+                <VideoEmbed
+                    nodeData={{
+                        argument: [{ value: video.id }],
+                        name: 'twitch',
+                    }}
+                />
+            </Modal>
+            <H5>{video.title}</H5>
+        </ThumbnailCard>
+    );
+};
+
 export default () => {
-    // TODO enable for launch
-    // const { error, live, pending, videos } = useTwitchApi();
+    const { error, stream, pending, videos } = useTwitchApi();
+    const twitchVideo = useMemo(() => {
+        if (stream) return stream;
+        if (videos && videos.length) return videos[0];
+        return null;
+    }, [stream, videos]);
+
     return (
         <Layout>
             <BackgroundImage>
                 {/* 
                 // TODO enable for launch
```

Comment:
Sorry, I misunderstood early, you can uncomment this now since we're making the API call anyway